client/app/components/parameters.html

```diff
@@ -1,7 +1,15 @@
-<div class="form-inline bg-white p-5" ng-if="parameters | notEmpty" ui-sortable="{ 'ui-floating': true, 'disabled': !editable }" ng-model="parameters">
-  <div class="form-group" ng-repeat="param in parameters">
+<div class="form-inline bg-white p-5"
+     ng-if="parameters | notEmpty"
+     ui-sortable="{ 'ui-floating': true, 'disabled': !editable }"
+     ng-model="parameters">
+  <div class="form-group m-l-10 m-r-10"
```

Comment:
Why add margin on both sides? Shouldn't m-r be enough?

Btw, did you  try putting the labels on top of the input boxes?